src/20240122_AaveV3Ethereum_RegisterADIScrollAdapter/AaveV3Ethereum_RegisterADIScrollAdapter_20240122.t.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveV3Ethereum_RegisterADIScrollAdapter_20240122, ICrossChainForwarder} from './AaveV3Ethereum_RegisterADIScrollAdapter_20240122.sol';
+import {GovernanceV3Scroll} from 'aave-address-book/GovernanceV3Scroll.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {ChainIds} from 'aave-helpers/ChainIds.sol';
+
+interface IBaseAdapter {
+  /**
+   * @notice method to get the trusted remote address from a specified chain id
+   * @param chainId id of the chain from where to get the trusted remote
+   * @return address of the trusted remote
+   */
+  function getTrustedRemoteByChainId(uint256 chainId) external view returns (address);
+}
+
+interface ICrossChainReceiver {
+  /**
+   * @notice method to get the current allowed receiver bridge adapters for a chain
+   * @param chainId id of the chain to get the allowed bridge adapter list
+   * @return the list of allowed bridge adapters
+   */
+  function getReceiverBridgeAdaptersByChain(
+    uint256 chainId
+  ) external view returns (address[] memory);
+}
+
+/**
+ * @dev Test for AaveV3Ethereum_RegisterADIScrollAdapter_20240122
+ * command: make test-contract filter=AaveV3Ethereum_RegisterADIScrollAdapter_20240122
+ */
+contract AaveV3Ethereum_RegisterADIScrollAdapter_20240122_Test is ProtocolV3TestBase {
+  AaveV3Ethereum_RegisterADIScrollAdapter_20240122 internal proposal;
+
+  uint256 public ethFork;
+  uint256 public scrollFork;
+
+  function setUp() public {
+    ethFork = vm.createFork(vm.rpcUrl('mainnet'), 19062921);
+    scrollFork = vm.createFork(vm.rpcUrl('scroll'), 2679843);
+
+    vm.selectFork(ethFork);
+    proposal = new AaveV3Ethereum_RegisterADIScrollAdapter_20240122();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
```

Comment:
Should we test all the configurations here @sendra ? Or the tests on the a.DI repo for Scroll are enough?

Reply:
tests for ccc configs on scroll, and the adapter registered there are included here already. added also one for confirmations